### PR TITLE
Fix sanitize media

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

Fix bug in sanitize function for medias

### Why is it needed?

Redacted for security reasons.

### How to test it?

DM if you need instruction

### CODE REVIEW

File: packages/core/strapi/tests/api/populate/sanitize.test.e2e.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+const schemas = {
+  contentTypes: {
+    a: {
+      kind: 'collectionType',
+      displayName: 'a',
+      singularName: 'a',
+      pluralName: 'as',
+      attributes: {
+        cover: { type: 'media', single: true },
```
Comment (soupette) [inline packages/core/strapi/tests/api/populate/sanitize.test.e2e.js:26]: I guess `single: true` works the same way as `multiple: false` in your test?

Comment (alexandrebodin) [reply packages/core/strapi/tests/api/populate/sanitize.test.e2e.js:26]: Actually that's a mistake

**Review by Convly — APPROVED** [review]
LGTM,
On a side note, I think we should prefer using internal services instead of a content API call when dealing with fixtures (such as the createFile). Here it's not an issue since we only care about the file ID, but it could have an impact when manipulating regular entities that have middlewares/policies or even specific validations. WDYT?

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Pull request merged**